assets/js/theme/common/product-details.js

```diff
@@ -595,4 +596,23 @@ export default class ProductDetails {
             $radio.attr('data-state', $radio.prop('checked'));
         });
     }
+
+    /**
+     * Check for fragment identifier in URL requesting a specific tab
+     */
+    getTabRequests() {
+        if (window.location.hash && window.location.hash.indexOf('#tab-') === 0) {
+            const $activeTab = $('.tabs').has(`[href='${window.location.hash}']`);
+            const $tabContent = $(`${window.location.hash}`);
+
+            $activeTab.find('.tab')
```

Comment:
Should we have a guard in case someone uses a url fragment that doesn't actually match a tab element on the page?